libs/domains/services/feature/src/lib/dockerfile-settings/dockerfile-settings.tsx

```diff
@@ -78,16 +77,22 @@ export function DockerfileSettings({ children, methods, onSubmit, directSubmit =
         handleSubmit()
       }}
     >
-      <InputSelect
-        label="File source"
-        options={availableFileSources}
-        value={fileSource}
-        onChange={(value) => {
-          setFileSource(value as FileSource)
-        }}
+      <Controller
+        name="dockerfile_source"
+        control={methods.control}
+        defaultValue="GIT_REPOSITORY"
+        render={({ field }) => (
+          <InputSelect
+            label="File source"
+            value={field.value as string}
```

Comment:
`as string` not necessary no ?

Reply:
Fixed with https://github.com/Qovery/console/pull/1459/commits/bd2624514654845e05ebb145d89ec34f2ebd9036